### PR TITLE
Features/rwhite/EMBCESSMOD-4909 - Responders - Needs Changes

### PR DESCRIPTION
**Pull request by williamrosswhite**
EMBCESSMOD-4909

- UI in Needs section of Responders side was updated to have checkboxes instead of radio buttons
- Shelter option was changed to provide two radio button options.

### CODE REVIEW

File: responders/src/UI/embc-responder/src/assets/images/wizard/does-not-need-support.svg
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/assets/images/wizard/does-not-need-support.svg:1]: Not sure if we want the default size to be that large

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/needs/needs.component.ts [hunk 1]
```diff
@@ -121,36 +148,161 @@ export class NeedsComponent implements OnInit, OnDestroy {
...
+    } else if (atLeastOneOtherCheckboxChecked) {
+      if (canEvacueeProvideLodging && !lodgingOptions) {
+        this.stepEssFileService.setTabStatus('needs', 'incomplete');
+      } else {
+        this.stepEssFileService.setTabStatus('needs', 'complete');
+      }
     } else {
       this.stepEssFileService.setTabStatus('needs', 'not-started');
     }
-    this.saveFormData();
```
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/needs/needs.component.ts:135]: I think this is still needed

File: responders/src/UI/embc-responder/src/assets/images/wizard/does-not-need-support.svg
Comment (williamrosswhite) [reply responders/src/UI/embc-responder/src/assets/images/wizard/does-not-need-support.svg:1]: this is listed as outdated, do we still want to address this?  I assume I could just scale it down

Comment (williamrosswhite) [reply responders/src/UI/embc-responder/src/assets/images/wizard/does-not-need-support.svg:1]: i resized it to match the size of the others

**Pull request merged**